core-rust/mesh-api-server/mesh-cli-configs/default.json

```diff
@@ -46,7 +46,7 @@
         }
     },
     "construction": {
-        "offline_url": "http://127.0.0.1:3337/mesh",
+        "offline_url": "http://localhost:3337/mesh",
```

Comment:
This is to cover the test requirement:
`offline_url should be different from offline_url and not empty`

Reply:
Cheeky :-D